R/rdeephaven/inst/tests/testthat/test_client_wrapper.R

```diff
@@ -0,0 +1,289 @@
+library(testthat)
+library(rdeephaven)
+
+setup <- function() {
+  df1 <- data.frame(
+    string_col = c("I", "am", "a", "string", "column"),
+    int_col = c(0, 1, 2, 3, 4),
+    dbl_col = c(1.65, 3.1234, 100000.5, 543.234567, 0.00)
+  )
+
+  df2 <- data.frame(
+    col1 = rep(3.14, 100),
+    col2 = rep("hello!", 100),
+    col3 = rnorm(100)
+  )
+
+  df3 <- data.frame(matrix(rnorm(10 * 1000), nrow = 10))
+
+  df4 <- data.frame(
+    time_col = seq.POSIXt(as.POSIXct(Sys.Date()), as.POSIXct(Sys.Date() + 30), by = "1 sec")[250000],
+    bool_col = sample(c(TRUE, FALSE), 250000, TRUE),
+    int_col = sample(0:10000, 250000, TRUE)
+  )
+
+  return(list("df1" = df1, "df2" = df2, "df3" = df3, "df4" = df4))
+}
+
+##### TESTING GOOD INPUTS #####
+
+test_that("client dhConnection works in the simple case of anonymous authentication", {
+
+  # TODO: assumes server is actually running on localhost:10000, this is probably bad for CI
```

Comment:
Should this be a todo or just a comment?  Should this attempt to pull the target from an env variable and then just default to the localhost string to make CI config easier?

If these changes are made, they may need to happen in all test files, so I suggest a utility function.

Optional

Reply:
I think this whole thing should be revisited when CI gets hooked up. I don't see the point in spending time guessing what that is going to require now.